src/test/relation_engine_client/test_integration.py

```diff
@@ -0,0 +1,137 @@
+        _id = self._save_test_vert()
+        bind_vars = {'key': _id}
+        qname = 'fetch_test_vertex'
+        result = self.client.stored_query(qname, bind_vars)
+        self.assertEqual(result['count'], 1)
+        self.assertEqual(result['results'][0]['_key'], _id)
+
+    def test_stored_query_invalid_args(self):
+        with self.assertRaises(TypeError):
+            self.client.stored_query()
+        with self.assertRaises(TypeError):
+            self.client.stored_query(123, 123)
+        with self.assertRaises(TypeError):
+            self.client.stored_query("")
+
+    def test_stored_query_unknown_query(self):
+        qname = 'xyz123'
+        with self.assertRaises(RERequestError) as ctx:
+            self.client.admin_query(qname, bind_vars={'key': 0})
+        self.assertEqual(ctx.exception.resp.status_code, 400)
+
+    def test_stored_query_missing_bind_vars(self):
+        qname = 'fetch_test_vertex'
+        with self.assertRaises(RERequestError) as ctx:
+            self.client.admin_query(qname, bind_vars={'x': 'y'})
+        self.assertEqual(ctx.exception.resp.status_code, 400)
+
+    def test_stored_query_raise_not_found(self):
+        _id = str(uuid4())
+        bind_vars = {'key': _id}
+        qname = 'fetch_test_vertex'
+        with self.assertRaises(RENotFound) as ctx:
+            self.client.stored_query(qname, bind_vars, raise_not_found=True)
+        self.assertTrue(_id in ctx.exception.req_body)
+
+    def test_save_docs_ok(self):
+        _id = str(uuid4())
+        docs = [{'_key': _id}]
+        results = self.client.save_docs(coll=_VERT_COLL, docs=docs)
+        self.assertEqual(results['created'], 1)
+        self.assertFalse(results['error'])
+        self.assertEqual(results['errors'], 0)
+        self.assertEqual(results['ignored'], 0)
+        self.assertEqual(results['updated'], 0)
+
+    def test_save_docs_invalid_args(self):
+        with self.assertRaises(TypeError):
+            self.client.save_docs()
+        with self.assertRaises(TypeError):
+            self.client.save_docs(123, 456)
+        # Empty docs list
+        with self.assertRaises(TypeError):
+            self.client.save_docs(_VERT_COLL, [])
+
+    def test_save_docs_unknown_coll(self):
+        with self.assertRaises(RERequestError) as ctx:
+            self.client.save_docs('xyz123', [{'_key': 0}])
+        self.assertEqual(ctx.exception.resp.status_code, 400)
+
+    def test_save_docs_invalid_docs(self):
+        with self.assertRaises(RERequestError) as ctx:
+            self.client.save_docs(_VERT_COLL, [{'hi': 0}])
+        self.assertEqual(ctx.exception.resp.status_code, 400)
+
+    # -- Test helpers
+
+    def _save_test_vert(self):
+        """Create a test vertex with a random & unique id."""
+        _id = str(uuid4())
+        docs = [{'_key': _id}]
+        results = self.client.save_docs(coll=_VERT_COLL, docs=docs)
+        if results['error']:
+            raise RuntimeError(results)
+        return _id
```

Comment:
What's the tar.gz below for?

Reply:
It's a cached version of the relation_engine_spec release for testing purpose, mainly so it can run on Travis without a Github API usage limit error getting thrown.

Reply:
A 1 sentence readme explaining that next to the file would be helpful for the person who inherits this repo

Reply:
:+1:

Reply:
Added description in README.md in this directory